src/components/toolbar/Navbar.tsx

```
@@ -71,58 +58,20 @@ export default function Navbar() {
-                {/* Logo or Home Link */}
-                <div className="flex items-center gap-4">
-                    <Link href="/" className="text-lg font-bold text-primary">
+            <div className="mx-auto flex px-4 py-2 justify-between items-center h-full border-b border-transparent bg-gradient-to-r from-background/70 via-background/40 to-background/70 dark:from-slate-950/60 dark:via-slate-900/40 dark:to-slate-950/60 backdrop-blur-md supports-[backdrop-filter]:bg-background/40">
+                <div className="flex items-center gap-3">
+                    <Link
+                        href="/"
+                        className="text-lg font-bold tracking-tight bg-gradient-to-r from-indigo-600 via-sky-600 to-cyan-600 dark:from-indigo-300 dark:via-sky-300 dark:to-cyan-300 bg-clip-text text-transparent"
+                    >
                         Dozu
                     </Link>
                     <ShowIf when={isStudent} children={<LearningModeSelect />} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)